Jenkinsfile

```diff
@@ -7,7 +7,7 @@
  - Run Python 3 unit tests in tox
  - Build Debian packages for supported Ubuntu versions
 
- If these steps succeed and the master branch is being built, wheels and debs are uploaded to PyPI and the
+ If these steps succeed and the master branch is being built, wheels and debs are uploaded to Artifactory and the
```

Comment:
Is there a merge issue here? We certainly were uploading this one to PyPI... Similar issues elsewhere in the file